projects/core/directives/dropdown/dropdown-selection.directive.ts

```diff
@@ -167,8 +167,9 @@ export class TuiDropdownSelectionDirective
         ghost.style.height = tuiPx(height);
         ghost.textContent = CHAR_ZERO_WIDTH_SPACE + value + CHAR_NO_BREAK_SPACE;
 
-        range.setStart(ghost.firstChild!, selectionStart || 0);
-        range.setEnd(ghost.firstChild!, selectionEnd || 0);
+        // TODO: improve typing later (ChildNode | null)
```

Comment:
I think this TODO is unnecessary, there's nothing we can improve and we just created that `firstChild` as a text node one line earlier.

Reply:
fixed